jujugui/static/gui/src/app/components/user-profile/model-list/model-list.js

```diff
@@ -83,10 +86,8 @@ class UserProfileModelList extends React.Component {
           return model.owner === extUser;
         });
       }
-      if (modelList.length) {
-        broadcastStatus('ok');
-      } else {
-        broadcastStatus('empty');
+      if (modelList && modelList.length > 0) {
+        this.props.setHasEntities(true);
```

Comment:
Do we ever set this to false? I mean, even if we do, I am not sure about one single component setting has entities to falso for all the others. So maybe simplify this call like `this.props.notifyHasEntities()` ?